modules/core/src/test/java/org/apache/ignite/testframework/junits/GridAbstractTest.java

```diff
@@ -1739,7 +1739,9 @@ protected void stopAndCancelGrid(int idx) {
     /**
      * @return Grid test configuration.
      * @throws Exception If failed.
+     * Don't use this method in test, it doesn't call.
      */
+    @Deprecated
```

Comment:
I think this annotation is too restrictive. Let's add a message to the Javadoc.
For instance: `Be aware that this method is not called by startGrid(int idx)`